server/GQLFactory/helpers/typeHelpers.js

```diff
@@ -73,10 +73,47 @@ mutationsHelper.mutationFields = (
 
 /*   Functions facilitating creation of custom types */
 const customHelper = {};
+/*  Loops through SQL columns getting their name as fieldName, type, and isNullable to be returned as fields */
+customHelper.getFields = (primaryKey, foreignKeys, columns) => {
+  let fields = ``;
 
-customHelper.getFields = () => {};
+  for (const fieldName of Object.keys(columns)) {
```

Comment:
can we add a comment here to clarify what the conditional is filtering for?

Reply:
going to change syntax as well !foreignKeys || !foreignKeys[fieldName]...